com.github.christianrauch.Jahresarbeit-2003.yaml

```
@@ -0,0 +1,121 @@
+app-id: com.github.christianrauch.Jahresarbeit-2003
+runtime: org.freedesktop.Platform
+runtime-version: "21.08"
+sdk: org.freedesktop.Sdk
+command: /app/ja2003
+finish-args:
+- --socket=wayland
+- --socket=fallback-x11
+- --device=dri
```

```suggestion
- --device=dri
- --share=ipc
```

this is needed for X11 anyway.